1Password/op-scim-gcp-marketplace
# Move to use op-scim-helm chart

In this PR we update the GCP marketplace configuration to use the chart defined in [1Password/op-scim-helm](https://github.com/1Password/op-scim-helm).

The most significant changes are related to no longer duplicating the Helm chart configuration from `1Password/op-scim-helm`. We are instead depending on this chart by creating a [wrapper chart](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/building-deployer-helm.md#part-1b-optional-creating-a-wrapper-chart). With this in mind we removed much of the logic involved in creating custom SCIM bridge and redis images.

We have also updated the project and chart `README.md` files.

To test:
1. Make sure you are in the `op-scim-bridge` folder (same as `Makefile`)
2. Create an image using a test tag: `REGISTRY="gcr.io/op-scim-bridge" TAG=t<test-tag-name> make app/build`
3. Verify the test deployer: `TAG=<test-tag-name> make app/test`
4. Ensure that the verify succeeds
6. Install the test deployer on a GCP cluster:
     ```
    mpdev install --deployer=gcr.io/op-scim-bridge/op-scim-bridge/deployer:<test-tag-name> --parameters='{"name": "op-scim-bridge", "namespace": "op-scim-bridge", "accountDomain": "testing.1password.com", "scim.serviceAccount": "op-scim-bridge-deployer-sa" }'
    ```
7. Ensure that the install succeeds and that the application (and dependencies) are running without error

Also please review that the instructions in the updated `README.md` files to see if they are clear.
